odoo/addons/base/models/res_company.py

```diff
@@ -37,7 +37,7 @@ def _get_default_favicon(self, original=False):
                 return base64.b64encode(f.read())
             # Modify the source image to add a colored bar on the bottom
             # This could seem overkill to modify the pixels 1 by 1, but
-            # Pillow doesn't provide an easy way to do it, and this 
+            # Pillow doesn't provide an easy way to do it, and this
```

Comment:
I personnally don't care about this kind of "useless" diff, but not everyone in Odoo feels the same. In Account, we do what we want, but here, it's base. Could you set it back ?